.github/workflows/main.yml

```diff
@@ -43,3 +43,18 @@ jobs:
              huion-switcher
              README.md
              *.rules
+
+  create-release:
+    runs-on: ubuntu-latest
```

Comment:
I wonder if we should not pin an older ubuntu/debian/fedora release, so that the included libc is compatible with everything. For `udev-hid-bpf` I'm still using the EOL fedora 37 for this reason

Reply:
turns out ubuntu-latest is still 22.04 anyway which should be old enough. I'll change that to be explicit, thanks.